src/Core/maps/src/Handlers/Map/MapHandler.Android.cs

```diff
@@ -304,9 +282,20 @@ internal void OnMapReady(GoogleMap map)
 			Map = map;
 
 			map.SetOnCameraMoveListener(_mapReady);
+
 			map.MarkerClick += OnMarkerClick;
 			map.InfoWindowClick += OnInfoWindowClick;
 			map.MapClick += OnMapClick;
+
+			if (VirtualView != null)
```

Comment:
This is the main change. The mapper invoke the mapping methods to update basic properties, but in the case of the zoom for example, the map is not initialized yet and nothing is done. For that reason, we apply changes to set the basic properties when having the map ready.